localstack/services/apigateway/helpers.py

```diff
@@ -1330,6 +1330,13 @@ def get_event_request_context(invocation_context: ApiInvocationContext):
         request_context["authorizer"] = authorizer_result
     request_context["identity"].update(invocation_context.auth_identity or {})
 
+    # https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-override-request-response-parameters.html
```

Comment:
basic `requestOverride` implementation. Here we are allowing for the rendering namespace that will be further computed inside airspeed.